src/main/webapp/WEB-INF/templates/user_dashboard.jsp

```diff
@@ -0,0 +1,39 @@
+<%@page contentType="text/html" pageEncoding="UTF-8" %>
+<%@taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+
+<!DOCTYPE html>
+<html>
+<head>
+  <div class="heading">
+    <a class="logo" href="/">
+      <h1>
+        <span class="blue">S</span><span class="red">n</span><span class="yellow">i</span><span
+          class="blue">p</span><span class="green">p</span><span class="red">e</span><span
+          class="yellow">t</span>
+      </h1>
+    </a>
+  </div>
+  <meta charset="utf-8">
+  <title>User Dashboard</title>
+  <link rel="stylesheet" href="css/style.css">
+  <link rel="stylesheet" href="css/user_style.css">
+</head>
+<body>
+    <form class="setting" action="/user" method="POST">
+      <div class="row">
+        <label for="input-websites">Choose your preferred website:</label>
+        <select class="input-websites" id="input-websites" name="website">
+          <option value="geeksforgeeks">GeeksForGeeks</option>
+          <option value="stackoverflow">StackOverflow</option>
+          <option value="w3schools">W3Schools</option>
+        </select>
+      </div>
+      <div class="row">
+        <label for="input-languages">Choose your primary coding language:</label>
+        <input class="input-languages" id="input-languages" type="text" name="language"
+               placeholder="Input your primary coding language">
```

Comment:
It seems like the bottom language text input width is larger than the website select. Could you take a look?

Reply:
i'm not sure whether i used the correct approach to solve this issue..

Reply:
i added `width: -webkit-fill-available` to both input field

Reply:
No, anything prefixed with -webkit- is a hack that only works in Chrome and similar Chrome-like browsers. Avoid those except when absolutely necessary.

The crux of your issue is that &lt;select&gt; and &lt;input&gt; behave differently in their `box-sizing` property. &lt;select&gt; is `border-box`, which means its width includes padding and border, whereas &lt;input&gt; is `content-box`, which means width does not include padding and border.

Since your div gets a max-width of 400px, the select element gives its content only 378px so that the remaining 22px is allocated to its 10px padding and 1px border (on each side). The input element instead gives 400px to its content, but the extra 22px needed by the padding and border given causes it to go past the 400px max you've given its container div.

To fix, simply have `.input-languages` have `box-sizing: border-box;`.